src/processor_x86.cpp

```diff
@@ -715,10 +715,20 @@ static uint32_t sysimg_init_cb(const void *id)
     // We translate `generic` to `pentium4` or `x86-64` before sending it to LLVM
     // (see `get_llvm_target_noext`) which will be serialized into the sysimg target data.
     // Translate them back so we can actually match them.
+    bool sysimg_allows_no_cx16 = false;
```

Comment:
Maybe set this to `true` on 32bits (just replace `false` with `sizeof(void*) == 4`) so that the edge case of user explicitly specifying `+cx16` on 32bit will always be ignored. There's little reason for the user to specify that though so not a major issue...